crates/uv/tests/snapshots/ecosystem__warehouse-lock-file.snap

```diff
@@ -3144,7 +3144,7 @@ name = "redis"
 version = "5.0.8"
 source = { registry = "https://pypi.org/simple" }
 dependencies = [
-    { name = "async-timeout", marker = "python_full_version < '3.11.[X]'" },
+    { name = "async-timeout", marker = "python_full_version < '3.11.[X]' and python_version < '3.12'" },
```

Comment:
This looks wrong. I don't know where it's coming from.

Reply:
Debugging this.

Reply:
I haven’t figured this out yet, I think I should understand why this is happening before we merge @BurntSushi though you’re welcome to cherry-pick.

Reply:
Oh... I guess this is actually fine.

Reply:
It's just redundant.

Reply:
I needed to see it with fresh eyes.